certora/funnels/automation/StableSwapper.spec

```diff
@@ -0,0 +1,328 @@
+// StableSwapper.spec
```

Comment:
If you wanted to add a high-level invariant quantified over all functions here, something like `zzz <= block.timestamp` would be a good one that should hold. Should be covered by the existing rules, ofc.

Reply:
Yeah let's leave it for now as it is, excepting you see a very strong reason to add a general one, it should be enough with the cover per rule.